workers_launch_template.tf

```diff
@@ -282,6 +282,18 @@ resource "aws_launch_template" "workers_launch_template" {
     "instance_type",
     local.workers_group_defaults["instance_type"],
   )
+
+  dynamic "elastic_inference_accelerator" {
+    for_each = lookup(
+      var.worker_groups_launch_template[count.index],
+      "elastic_inference_accelerator",
+      local.workers_group_defaults["elastic_inference_accelerator"]
+    ) != null ? [lookup(var.worker_groups_launch_template[count.index], "elastic_inference_accelerator", local.workers_group_defaults["elastic_inference_accelerator"])] : []
+    content {
+      type = elastic_inference_accelerator.value
+    }
+  }
+
```

Comment:
Can't we just write something like this (because the default `local.workers_group_defaults["elastic_inference_accelerator"]` is `null`) ?

```hcl
"elastic_inference_accelerator" {
    type = lookup(
      var.worker_groups_launch_template[count.index],
      "elastic_inference_accelerator",
      local.workers_group_defaults["elastic_inference_accelerator"]
    )
}
```

Reply:
@barryib The  "elastic_inference_accelerator" block needs to be added only if the type has been specified. Otherwise we would get a block like: 
```
"elastic_inference_accelerator" {
     type = null
}
```
And this results in an error.